src/containers/DateTime.js

```diff
@@ -39,14 +40,33 @@ export function DateTime(props) {
 		)
 	}
 
-	const m = isNaN(props.value) || props.value > 9999999999 ? moment(props.value) : moment(props.value * 1000);
-	m.locale(language.slice(0, 2));
-
+	useEffect(() => {
+		if (language && language !== "en") {
+			fetchLocale()
+		} else {
+			setLocale(undefined);
+		}
+	}, [language])
+
+	const fetchLocale = async () => {
+		// Don't remove comment inside import
+		// it's webpack's dynamic expression
+		const importedLocale = await import(
+			/* webpackMode: "lazy", webpackChunkName: "df-[index]" */
+			`date-fns/locale/${language}/index.js`
+		);
+		setLocale(importedLocale.default);
+	}	
+
+
+	const date = isNaN(props.value) ? format(parseISO(props.value), 'PPp', { locale: locale }) :
+		props.value > 9999999999 ? format(props.value, 'PPp', { locale: locale }) :
```

Comment:
@titkov49 @ateska What do you think about using `Pp` as a time format? To me it looks a bit more convenient, when there are just numbers as a date. https://date-fns.org/v2.28.0/docs/format

Reply:
Ok, let's try that and we will quickly see.